### PR TITLE
[NeuralChat] Record user requests into DB

### PR DESCRIPTION
**Pull request by letonghan**
## Type of Change

feature
API not changed.

## Description

Add record_requests function in order to store user requests into MySQL database.
JIRA ticket: [JIRA 984](https://jira.devtools.intel.com/browse/NLPTOOLKIU-984)

## Expected Behavior & Potential Risk

User requests could be stored into DB using this util function.

## How has this PR been tested?

Local tested.

## Dependency Change?

no.

### CODE REVIEW

File: intel_extension_for_transformers/neural_chat/utils/database/init_db_requests.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
```
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/utils/database/init_db_requests.sql:1]: Should we add license header?

Comment (letonghan) [reply intel_extension_for_transformers/neural_chat/utils/database/init_db_requests.sql:1]: license added.

File: intel_extension_for_transformers/neural_chat/server/restful/retrieval_api.py [hunk 2]
```diff
@@ -73,6 +74,12 @@ async def retrieval_endpoint(request: RetrievalRequest) -> RetrievalResponse:
 async def retrieval_upload(file: UploadFile = File(...)):
     global plugins
     filename = file.filename
+    try:
+        record_request(request_url="/v1/askdoc/upload",
+                    request_body={'filename': filename},
+                    user_id='default')
+    except Exception as e:
+        logger.error(f"[askdoc - upload] Fail to record request into db. {e}")
     path_prefix = "/home/sdp/askdoc_upload/enterprise_docs/"
```
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/server/restful/retrieval_api.py:83]: why hard-coded path sdp?

Comment (letonghan) [reply intel_extension_for_transformers/neural_chat/server/restful/retrieval_api.py:83]: I'll refine it.

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**